### DISCUSSION TITLE
Why So_h is commented in the WW3 cap?

### DISCUSSION THREAD

**Discussion by uturuncoglu** [NOAA-EMC/WW3#1280] [General]
We are plaining to use sea surface height for coupling with ocean model under UFS Coastal since it is important for the shallow regions but when I checked the the NUOPC cap (mesh cap) it seems that `So_h` variable is commented out. Is there any specific reason for it? Otherwise, it would be activate it and test it under UFS Coastal.

https://github.com/NOAA-EMC/WW3/blob/d9b3172f4197c65d471662c6952a668152d71230/model/src/wav_import_export.F90#L109

**Comment by uturuncoglu** [comment]
@JessicaMeixner-NOAA @DeniseWorthen Any idea?

**Reply by JessicaMeixner-NOAA** [reply]
I do not know.

**Comment by DeniseWorthen** [comment]
@uturuncoglu I know of no specific reason it is not enabled. We do not export ``So_h`` from MOM6 (only the slopes), which are used by CICE. I can only presume that ``So_h`` coupling was never considered important for the global scales.

**Comment by uturuncoglu** [comment]
Okay. Let me try in my end to see what happens. If it works without any issue. Maybe we could activate it. It would be nice to ask also to @mvertens since it is part of NCAR cap.

**Reply by aliabdolali** [reply]
the only possible issue is with the mediator. We should pick proper interpolation in the mediator, otherwise we might run into wrong field distribution from circulation to wave and vice versa. Your simulation might fail if large/unrealistic radiation stress from wave and current field from circulation model got exchanged there.
The wave model decomposes all the wet and dry elements, so it should work stand alone with wetting and drying.

**Comment by aliabdolali** [comment]
@uturuncoglu It is due to scale and coarseness of global scale applications (~km), which were the initial intention of UFS-weather-model development where the wave component runs on static depth (MSL). The MOM6 grid is also curvilinear (I recall it around 5 to 10 min, which is very coarse in the coastal regions). The topobathy is not resolved in such scale, therefore water surface was excluded. I did some preliminary work, showing it does not add values on waves for such coarse resolution and also in the ocean model. On the contrary, in coastal scale, water level and current in very shallow water matter (tens of meter scale). 
Initially,

**Reply by uturuncoglu** [reply]
@aliabdolali @JessicaMeixner-NOAA @DeniseWorthen Thanks to all. Let me test to see how it works. I'll update you.
